WcaOnRails/app/models/regulation.rb

```diff
@@ -0,0 +1,16 @@
+class Regulation < SimpleDelegator
+  begin
+    REGULATIONS = JSON.parse(File.read(Rails.root.to_s + "/app/views/search_results/wca-regulations.json")).freeze
```

Comment:
Hmm right, so catching the exception as it is (assumming that we won't need REGULATIONS in the future) seems ok.
If we will need REGULATIONS (perhaps in specs) then it's not enough to allow travis to have them because we run specs locally. But that's not the case at the moment =)